card/lib/card/query.rb

```diff
@@ -46,25 +45,26 @@ class Query
     include Attributes
 
     ATTRIBUTES = {
-      basic:           %w{ name type_id content id key updater_id left_id
-                           right_id creator_id updater_id codename },
-      relational:      %w{ type part left right editor_of edited_by
-                           last_editor_of last_edited_by creator_of created_by
-                           member_of member },
-      plus_relational: %w{ plus left_plus right_plus },
-      ref_relational:  %w{ refer_to referred_to_by link_to linked_to_by include
-                           included_by },
-      conjunction:     %w{ and or all any },
-      special:         %w{ found_by not sort match complete extension_type },
-      ignore:          %w{ prepend append view params vars size }
-    }.inject({}) {|h,pair| pair[1].each {|v| h[v.to_sym]=pair[0] }; h }
+      basic:           %w( id name key type_id content left_id right_id
+                           creator_id updater_id codename                     ),
+      relational:      %w( type part left right
+                           editor_of edited_by last_editor_of last_edited_by
+                           creator_of created_by member_of member             ),
+      plus_relational: %w( plus left_plus right_plus                          ),
+      ref_relational:  %w( refer_to referred_to_by
+                           link_to linked_to_by
+                           include included_by                                ),
+      conjunction:     %w( and or all any                                     ),
+      special:         %w( found_by not sort match complete extension_type    ),
+      ignore:          %w( prepend append view params vars size )
+    }.inject({}) {|h,pair| pair[1].each { |v| h[v.to_sym]=pair[0] }; h }
```

Comment:
Do not use semicolons to terminate expressions.<br>Space missing after comma.<br>Surrounding space missing for operator '='.<br>Space between { and | missing.